api/pipeline.yaml

```diff
@@ -91,7 +124,7 @@ spec:
               printf no | tee /tekton/results/output-result
             fi
       runAfter:
-        - build-push-image
+        - create-new-migration-job
```

Comment:
We will be needing both. What if the buildah task for API image build fails?

Reply:
okay

Reply:
updated